### PR TITLE
Conviva Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

https://tv.yousee.dk/login
https://www.sonyliv.com/
https://www.factoriadeficcion.com/
https://www.xploretv.at/login
https://xploretv.si/setup?step=language
https://www.yasss.es/

### CODE REVIEW

File: src/technologies/c.json [hunk 1]
```diff
@@ -3142,6 +3142,19 @@
     "scriptSrc": "cdn\\.conveythis\\.com",
     "website": "https://www.conveythis.com/"
   },
+  "Conviva": {
+    "cats": [
+      14
+    ],
+    "description": "Conviva is a streaming partner & a single source of truth of streaming analytics for your entire team.",
+    "icon": "conviva.png",
```
Comment (nurbek91) [inline src/technologies/c.json:3150]: Conviva
svg available mate

File: src/technologies/c.json [hunk 2]
```diff
@@ -3142,6 +3142,19 @@
     "scriptSrc": "cdn\\.conveythis\\.com",
     "website": "https://www.conveythis.com/"
   },
+  "Conviva": {
+    "cats": [
+      14
+    ],
+    "description": "Conviva is a streaming partner & a single source of truth of streaming analytics for your entire team.",
```
Comment (nurbek91) [inline src/technologies/c.json:3149]: Conviva is a census, continuous measurement and engagement platform for streaming media.

File: src/technologies/c.json [hunk 3]
```diff
@@ -3142,6 +3142,19 @@
     "scriptSrc": "cdn\\.conveythis\\.com",
     "website": "https://www.conveythis.com/"
   },
+  "Conviva": {
+    "cats": [
+      14
```
Comment (nurbek91) [inline src/technologies/c.json:3147]: 103
14

File: src/technologies/c.json [hunk 4]
```diff
@@ -3142,6 +3142,19 @@
...
+    ],
+    "description": "Conviva is a streaming partner & a single source of truth of streaming analytics for your entire team.",
+    "icon": "conviva.png",
+    "js": {
+      "Conviva": "",
+      "Conviva.Client": "",
+      "Conviva.Client.version": "^([0-9.]+)?\\;version:\\1"
```
Comment (nurbek91) [inline src/technologies/c.json:3154]: `"Conviva.Client.version": "^([0-9\\.]+)$\\;version:\\1"`

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/c.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/c.json:3150]: from where you get the `SVG`s?

Comment (nurbek91) [reply src/technologies/c.json]: from website

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**